.github/workflows/build-release.yml

```diff
@@ -105,9 +105,38 @@ jobs:
           path: |
             build/linux/x64/release/bundle/*
 
+  build-macos:
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+
+      - name: Install CocoaPods
+        run: |
+          sudo gem install cocoapods
+
+      - name: Build macOS
+        run: |
+          flutter config --enable-macos-desktop
+          flutter pub get
+          cd macos && pod install && cd ..
```

Comment:
Instead of a cd macos ... cd ..,
use `pod install --project-directory=macos`
as mentioned in https://guides.cocoapods.org/terminal/commands.html#pod_install